packages/tools/replay-tool/src/replayLoaderObject.ts

```diff
@@ -47,8 +48,11 @@ export class ReplayCodeLoader implements ICodeLoader, IFluidCodeDetailsComparer
         return this;
     }
 
-    public async load(source: IFluidCodeDetails): Promise<IFluidModule> {
-        return Promise.resolve(this.fluidModule);
+    public async load(source: IFluidCodeDetails): Promise<IFluidModuleWithDetails> {
+        return {
+            module: this.fluidModule,
+            details: source,
```

Comment:
It seems `source` can be a string or an object based on logic above. Probably need some handling before being added to details